packages/ui/src/views/repo/repo-settings/repo-settings-page.tsx

```diff
@@ -0,0 +1,85 @@
+import { Outlet } from 'react-router-dom'
+
+import { SandboxLayout } from '@/views'
+import { MenuGroupTypes } from '@components/navbar/types'
+
+import { RepoSidebar } from './components/repo-sidebar'
+
+export const navItems = [
+  {
+    groupId: 0,
+    title: 'General',
+    type: MenuGroupTypes.GENERAL,
+
+    items: [{ id: 0, title: 'General Settings', to: 'general' }]
+  },
+  // {
```

Comment:
Do we need comments

Reply:
yeah, we can keep adding them back as the pages are made. Removed all dead links for now.